src/components/PerPageSelector/PerPageSelector.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './PerPageSelector.css';
+
+export const PerPageSelector = (props) => {
+  const { perPage, perPageOptions, onPerPageChange } = props;
+
+  const handlePerPageChange = (evt) => {
```

Comment:
event

Reply:
move this function to the place where you call it

Reply:
Discussed that on Slack. Done.